integration_tests/suite/base/test_devices.py

```diff
@@ -159,6 +159,24 @@ def test_list_unallocated(main, sub):
     assert_that(response.items, all_of(has_item(main), not_(has_item(sub))))
 
 
+@fixtures.device()
+@fixtures.device()
+def test_list_device_no_config(dev1, dev2):
+    real_device = provd.devices.get(dev1['id'])
+    real_device['config'] = None
+    provd.devices.update(real_device)
+
+    new_dev2 = confd.devices(dev2['id']).get().item
```

Comment:
Hi @bloom1
My morning brain refuses to understand what `new_dev2` is used for in the test. Is it different from `dev2`? Does GETing it tests a side effect of fixing the bug?